utils/auto_annotation/run_model.py

```diff
@@ -1,8 +1,8 @@
 import os
-import sys
 import json
 import argparse
-import traceback
+import random
+import logging
 
 os.environ['DJANGO_SETTINGS_MODULE'] = 'cvat.settings.production'
```

Comment:
@benhoff , one more request. Could you please remove dependency on Django in the script? Move common code in a file which doesn't have Django specific if it is possible. run_inference_engine_annotation should not depend on Django. Let me know if you think that you are not going to fix that now. The patch looks great for me.

Reply:
@nmanovic let's get this merged and then I'll work that.